packages/flutter/lib/src/rendering/paragraph.dart

```diff
@@ -835,7 +833,11 @@ class RenderParagraph extends RenderBox
         workingText,
         semanticsLabel: workingLabel,
       ));
-    } else {
+    }
+    // `workingText` has a non-nullable return type, but might be null when
+    // running with weak checking, so we need to null check it anyway (and
+    // ignore the warning that the null-handling logic is dead code).
```

Comment:
this comment is wrong, workingText can't be null in any mode. the code is just wrong (unrelated to this migration).